observability/how-to/activate-managed-alerts.mdx

```diff
@@ -0,0 +1,39 @@
+---
+meta: How to activate managed alerts
+  title: How to activate managed alerts
+  description: This page shows you how to activate managed alerts
+content:
+  h1: How to activate managed alerts
+  paragraph: This page shows you how to activate managed alerts
+categories:
+  - observability
+dates:
+  validation: 2022-10-13
+  posted: 2022-10-03
+---
+
+This page shows you how to activate [managed alerts](link) using the [Scaleway console](link).
```

Comment:
link to concept & then the console?

Reply:
Now that there's no more concept hover, I think maybe it would be a good idea to add a sentence explaining briefly why you might be interested in activating managed alerts and/or what they are.